build.gradle

```diff
@@ -11,10 +11,11 @@ buildscript {
         }
```

Comment:
delete this? per https://firebase.google.com/docs/crashlytics/upgrade-sdk?platform=android

Reply:
@bryanwussow Yes, sorry! I've been very busy with the presentation that I didn't look into this. I just saw that Chris fixed this. Thanks @chrisvire